src/components/SelectionList/Search/TransactionListItemRow.tsx

```diff
@@ -265,6 +265,11 @@ function TransactionListItemRow({
     const StyleUtils = useStyleUtils();
     const theme = useTheme();
 
+    const isOnHold = useMemo(() => TransactionUtils.isOnHold(item), [item]);
+
+    const isApproveAction = item.action === CONST.SEARCH.ACTION_TYPES.APPROVE;
+    const shouldUseSuccessStyle = isApproveAction ? !isOnHold : true;
```

Comment:
Do we need to check if it's approve action? Shouldn't we always use the grey style if there's a transaction on hold?

```suggestion
```